tools/cmake/modules/FindSDLSqlite3.cmake

```diff
@@ -0,0 +1,15 @@
+#  - Try to find Sqlite3
+#
+#  SQLITE3_INCLUDE_DIRECTORY - the Sqlite3 include directory
+#  SQLITE3_LIBRARIES - the Sqlite3 libraries
+
+get_os(OS)
+if(${OS} STREQUAL "posix")
+  find_package(Sqlite3 REQUIRED)
+elseif(${OS} STREQUAL "win")
```

Comment:
@istoilovgithub please remove win and qt dependencies. Qt and Win only in pre_5_0